platform/linux-dpdk/pool/dpdk.c

```diff
@@ -91,12 +90,12 @@ int odp_pool_init_global(void)
 	return 0;
 }
 
-int odp_pool_init_local(void)
+static int dpdk_pool_init_local(void)
```

Comment:
Minor issue: if it does nothing we can remove it and just set the function pointer to NULL.